kolibri/core/errorreports/models.py

```diff
@@ -47,25 +55,58 @@ def allow_migrate(self, db, app_label, model_name=None, **hints):
 
 
 class ErrorReports(models.Model):
-    error_from = models.CharField(max_length=10, choices=POSSIBLE_ERRORS)
+    category = models.CharField(max_length=10, choices=POSSIBLE_ERRORS)
     error_message = models.CharField(max_length=255)
     traceback = models.TextField()
     first_occurred = models.DateTimeField(default=timezone.now)
     last_occurred = models.DateTimeField(default=timezone.now)
-    sent = models.BooleanField(default=False)
-    no_of_errors = models.IntegerField(default=1)
+    reported = models.BooleanField(default=False)
+    events = models.IntegerField(default=1)
+    release_version = models.CharField(
```

Comment:
I would rather this was a longer CharField and that we reported the complete version string here.